docs/www/guide-wasm-elastic.md

```diff
@@ -0,0 +1,218 @@
+    going to handle that error, in this case, Deregister policy will
+    remove this wasm function from wasm engine.    
+  */
+});
+
+exports["default"] = transform;
+```
+
+after applying the new logic, we need to bundle everything into a single file.
+For it, we can use 
+
+```bash
+#this command generate `dist/elastic.js` file.
+$ npm run build
+```
+
+Before deploying this wasm function, we need to create the `produce` topic
+The statically defined input topic that this wasm operator has defined in its metadata
+
+```bash
+rpk topic create produce
+```
+
+## Deploy wasm function to Redpanda
+
+we use `rpk wasm deploy ~//wasm_example/wasm/dist/elastic.js` in this case
+
+```bash
+$ rpk wasm deploy dist/elastic.js
+
+Sent record to partition 0 at offset 1 with timestamp 2021-02-01 18:18:15.734185538 -0500 -05 m=+0.053943881.
+```
+
+To verify that the coprocessor has successfully been deployed, you can look through the redpanda logs for a log line that looks like [this]
+for check it.
+
+on `redpanda logs`
+
+```bash
+INFO [shard 0] coproc - service.cc:101 - Request recieved to register coprocessor with id: 14103244480447969041
+```
+
+The wasm engine itself also has a logfile which by default will exist in `/var/lib/redpanda/coprocessor/logs/wasm`. To see the status of your transform reported by the wasm engine look for log lines like [this]
+
+```bash
+[FileManager] info: Detected new file in submit dir: /var/lib/redpanda/coprocessor/submit/elastic.js
+
+[FileManager] info: Initiating RPC call to redpandas coprocessor service at endpoint - enable_coprocessors with data: [object Object]
+TODO: improve [object Object]
+```
+
+Run data through the coprocessor
+
+To transform data, push records onto the `produce` topic using any kafka client. In this example we will use the kafka client within `rpk`.
+
+```bash
+$ rpk topic produce produce -H logService:elastic -n 5
+```
+
+In `/var/lib/redpanda/coprocessor/logs/wasm` you should see log line that look like [this]
+
+```
+Applying wasm function
+Index record value to Elastic Search
+```
+
+## Verify Elastic Search results
+Now we need to validate that our wasm function, take a records and publish to ElasticSearch. To do this, we can use the same client for making a query to ElasticSearch, where we want to take all hist that has a wasm index, in this case `result_wasm`.
+
+```js
+const { Client } = require("@elastic/elasticsearch")
+
+const client = new Client({ node: 'http://localhost:9200' })
+
+client.search({
+  index: 'result_wasm'
+}).then((results) => {
+  const results = results.body.hits.hits.map(record => record._source)
+  console.log(results)
+})
+```
+ expected output:
+
+ ```bash
+ [
+  { record_message: 'Information from Redpanda + Wasm' },
+  { record_message: 'Information from Redpanda + Wasm' },
+  { record_message: 'Information from Redpanda + Wasm' },
+  { record_message: 'Information from Redpanda + Wasm' },
+  { record_message: 'Information from Redpanda + Wasm' }
+]
+ ```
```

Comment:
The guide needs an end so that the user knows it's finished and they can go on living their lives.

Reply:
Explain why this is the expected output.

Reply:
Done